File: python_t3_full/Clone_2083.py
def getPrimes(n) :
	yield 2
	i = 3
	while i < n :
		for a in getPrimes(int(math.sqrt(i)) + 1) :
			if i % a == 0 :
				break
		else :
			yield i
		i += 2


def getPrimes(n):
    if n > 2: yield 2
    i = 3
    while i < n:
        sqrt_i = int(math.sqrt(i)) + 1
        for a in getPrimes(sqrt_i):
            if i % a == 0 :
                break
        else:
            yield i
        i += 2


# 9

import math

File: java_t3_full/Clone_2083.java
public static double sqrt (int n) {
    double low = 0;
    double high = n;
    double mid = (high - low) / 2;
    while (Math.abs ((mid * mid) - n) > 0.000000000001) {
        if ((mid * mid) > n) {
            high = mid;
            mid = (high - low) / 2;
        } else {
            low = mid;
            mid = mid + ((high - low) / 2);
        }
    }
    return mid;
}


 public static double sqrt(int n) {
    double start = 0; 
    double end = n;
    double mid = (start + end) / 2;
    while (Math.abs(mid * mid - n) > 0.000000000001) { 
        if (mid * mid < n) {
            start = mid;
            mid = (start + end) / 2;
        }
        else { 
            end = mid;
            mid = (start + end) / 2;
        }
    }
    return mid;
}


